### PR TITLE
Merge panama-vector:master

### PR DESCRIPTION
**Pull request by sviswa7**
This pull request merges master changes into vectorIntrinsics.
Please review.

Best Regards,
Sandhya

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed

### Download
`$ git fetch https://git.openjdk.java.net/panama-vector pull/33/head:pull/33`
`$ git checkout pull/33`

### CODE REVIEW

File: test/hotspot/gtest/aarch64/asmtest.out.h [hunk 1]
```diff
@@ -540,131 +540,135 @@
     __ sminv(v19, __ T8H, v20);                        //       sminv   h19, v20.8H
```
Comment (nsjian) [inline test/hotspot/gtest/aarch64/asmtest.out.h:540]: A manual merge seems not working fine for this file to resolve conflicts. I will sent you the regenerated file.

**Review by nsjian — COMMENTED** [review]
Please help to update aarch64 asmtest.out.h, otherwise looks good to me.

My local tests for this merge passed, except for AArch64 GTestWrapper.java case, which was caused by the merge of asmtest.out.h.

File: test/hotspot/gtest/aarch64/asmtest.out.h (hunk 1 above)
Comment (sviswa7) [reply test/hotspot/gtest/aarch64/asmtest.out.h:540]: @nsjian Thanks a lot. I pushed the regenerated asmtest.out.h.

**Pull request closed without merging**